modules/mobile/app/serializers/mobile/v2/user_serializer.rb

```diff
@@ -8,17 +8,29 @@ class UserSerializer
       include JSONAPI::Serializer
 
       set_type :user
-      attributes :id, :first_name, :middle_name, :last_name, :signin_email, :birth_date, :signin_service
+      attributes :id, :first_name, :middle_name, :last_name, :signin_email, :birth_date, :signin_service,
+                 :has_facility_transitioning_to_cerner
 
       def initialize(user)
         @user = user
         birth_date = user.birth_date.nil? ? nil : Date.parse(user.birth_date).iso8601
         resource = UserStruct.new(user.uuid, user.first_name, user.middle_name, user.last_name, user.email,
-                                  birth_date, user.identity.sign_in[:service_name].remove('oauth_'))
+                                  birth_date, user.identity.sign_in[:service_name].remove('oauth_'),
+                                  transitioning_facility?(user))
         super(resource)
       end
 
-      UserStruct = Struct.new(:id, :first_name, :middle_name, :last_name, :signin_email, :birth_date, :signin_service)
+      # this is a temporary fix
+      def transitioning_facility?(user)
+        transitioning_facility_id = Rails.env.production? ? '556' : '459'
```

Comment:
I couldn't find any test users with facility 556 in staging, so I had to choose a different facility for staging.